install_mason.sh

```diff
@@ -14,9 +14,9 @@ source local.env
 
 if [ ! -d mason_packages ] ; then
 
-install boost 1.63.0
-install boost_libtest 1.63.0
-install boost_libiostreams 1.63.0
+install boost 1.73.0
```

Comment:
Are these upgrades required?  If not, I don't think we should do them.  We do statically link these libs, so it may not matter.

Reply:
Reverted